README.md

```diff
@@ -56,7 +57,7 @@ func main() {
     // Repair square.
     repaired, err := rsmt2d.RepairExtendedDataSquare(
         eds.RowRoots(),
-        eds.ColumnRoots(),
+        eds.ColRoots(),
```

Comment:
I actually liked the written-out `ColumnRoots` better. Especially for public APIs (for local vars, and internal methods col is just fine). But I guess that is a matter of taste and I can get used to Col. I like that you changed it consistently in all places (as far as I can see). I feel consistency is more important here.